.eslintrc.cjs

```diff
@@ -4,13 +4,15 @@ require('@rushstack/eslint-patch/modern-module-resolution')
 module.exports = {
   root: true,
   extends: [
-    'plugin:vue/vue3-essential',
```

Comment:
Why has this been removed?

Reply:
It's included in `plugin:vue/vue3-recommended`, ref: https://eslint.vuejs.org/user-guide/#bundle-configurations-eslintrc